src/requests/process-method.ts

```diff
@@ -89,10 +89,26 @@ function getParamSeparation(paramGroups: Dictionary<Parameter[]>): string[] {
       return def;
     }
 
-    // only one direct body parameter is allowed
-    // 1st one taken if more of them present
     if (groupName === 'body') {
-      def = `params.${group[0].name};`;
+      // when the schema: { '$ref': '#/definitions/exampleDto' } construct is used
+      if ('schema' in group[0]) {
```

Comment:
if the parameters contain "schema" attribute, only the "schema" attribute will be applied